binary_transparency/firmware/cmd/ft_personality/impl/ft_personality.go

```diff
@@ -33,8 +33,8 @@ import (
 	"github.com/gorilla/mux"
 	"golang.org/x/mod/sumdb/note"
 
-	_ "github.com/google/trillian/merkle/rfc6962/hasher" // Load hashers
-	_ "github.com/mattn/go-sqlite3"                      // Load drivers for sqlite3
+	_ "github.com/google/trillian/merkle/rfc6962" // Load hashers
```

Comment:
This is not needed as we don't have "registers" now?

Reply:
Good spot - these were all done mechanically with `gofmt -r`